test/dot_s/alias.wast

```diff
@@ -1,5 +1,5 @@
 (module
-  (memory 0)
+  (memory 1)
```

Comment:
There isn't a stack in this file, why are we adding memory? Seems like we shouldn't allocate the stack if there isn't one. So something is going wrong.